ChangeLog.rst

```diff
@@ -4,6 +4,7 @@ Release 3.5.1 (2018-12-22)
 * Documentation updates
 * Build system updates
 * Added "BindInterface" as valid "-o" option.
+* Fixed "-o idmap=user" to map both UID and GID on all OSs.
```

Comment:
3.5.1 is already released :-). You need to add it on top, for the next version.

Reply:
Oh gosh, yes, how silly of me. 😏